lint/lint.go

```diff
@@ -42,6 +43,14 @@ func Debug(debug bool) LinterOption {
 	}
 }
 
+// IgnoreCommentDisables sets the flag for ignoring
+// comments which disable rules.
```

Comment:
I think you can fit more words on the first line. I use `gq` in vim. Not sure what editor you use though.

Reply:
I prefer shorter lines, but it's not a strong opinion at all. I'll change it.